cmd/virt-launcher/virt-launcher.go

```diff
@@ -610,16 +615,31 @@ func RemoveContents(dir string) error {
 
 func terminateIstioProxy() {
 	if istioProxyPresent() {
-		resp, err := http.Post(fmt.Sprintf("http://localhost:%d/quitquitquit", infraconfigurators.EnvoyMergedPrometheusTelemetryPort), "", nil)
-		if err != nil || resp.StatusCode != http.StatusOK {
-			log.Log.Error("Failed to request Istio proxy termination")
+		backoff := wait.Backoff{
```

Comment:
I'm unsure, but I guess you just invented these numbers. I'd use one invented by someone else - https://github.com/kubernetes/client-go/blob/9781541482553fce5a4667b918a74b11d0816a33/util/retry/util.go#L38

Unless you actually know why these numbers are good, of course.

Reply:
Done